apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -0,0 +1,13 @@
+ack_generate_info:
+  build_date: "2022-06-30T13:57:00Z"
+  build_hash: 6acf40fe3e3cfd97b799ef7cbf1e89e01c3db8f7
+  go_version: go1.18.3
+  version: v0.18.4-15-g6acf40f-dirty
```

Comment:
looks like you do not have latest code-generator checked out. Please pull latest code-generator changes and regenerate your controller again.